CHANGELOG.md

```diff
@@ -4,6 +4,83 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [v0.1.0-preview.4] - 2022-04-11
+## ❗ BREAKING ❗
+- **Telemetry simplification** [PR #782](https://github.com/apollographql/router/pull/782)
+
+  Telemetry configuration has been reworked to focus exporters rather than OpenTelemetry. Users can focus on what they are tying to integrate with rather than the fact that OpenTelemetry is used in the Apollo Router under the hood.
```

Comment:
```suggestion
  Telemetry configuration has been reworked to focus exporters rather than OpenTelemetry. Users can focus on what they are trying to integrate with rather than the fact that OpenTelemetry is used in the Apollo Router under the hood.
```